webviews/components/timeline.tsx

```diff
@@ -55,6 +55,8 @@ const CommitEventView = (event: CommitEvent) =>
 			<a className='message' href={event.htmlUrl}>{event.message}</a>
 		</div>
 		<a className='sha' href={event.htmlUrl}>{event.sha.slice(0, 7)}</a>
+		{nbsp}
+		<div>{event.authoredDate}</div>
```

Comment:
There's a `Timestamp` component we could use here to show the date formatted as "X days ago" which I think is more consistent with how other dates are displayed

Reply:
Works great, thanks for the pointer!